src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+}
```

Comment:
Avoid using the 'align-items: center;' property on the .nav__item class. This property will vertically align the text inside the list items which might not be what you want. If you want to center the text horizontally, you can use 'text-align: center;' instead.